docs/samples/Microsoft.ML.AutoML.Samples/Program.cs

```diff
@@ -8,6 +8,7 @@ public static void Main(string[] args)
         {
             try
             {
+                Cifar10.Run();
```

Comment:
Don't forget to clear the console after this.

Reply:
Sure